File: SDKs/zoom.us/2.0.0/typescript/src/sdk/models/operations/deletevoicemail.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class DeleteVoicemailPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=voicemailId" })
  voicemailId: string;
}


export class DeleteVoicemailRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: DeleteVoicemailPathParams;
}


export class DeleteVoicemailResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  body?: Uint8Array;

  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  deleteVoicemail204ApplicationJSONAny?: any;
}

File: SDKs/zoom.us/2.0.0/python/src/sdk/models/operations/deletevoicemail.py
import dataclasses
from typing import Any,Optional


@dataclasses.dataclass
class DeleteVoicemailPathParams:
    voicemail_id: str = dataclasses.field(metadata={'path_param': { 'field_name': 'voicemailId', 'style': 'simple', 'explode': False }})
    

@dataclasses.dataclass
class DeleteVoicemailRequest:
    path_params: DeleteVoicemailPathParams = dataclasses.field()
    

@dataclasses.dataclass
class DeleteVoicemailResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    body: Optional[bytes] = dataclasses.field(default=None)
    delete_voicemail_204_application_json_any: Optional[Any] = dataclasses.field(default=None)
    
